item_test.go

```diff
@@ -93,3 +100,36 @@ func Test_Item_ExpiresAt(t *testing.T) {
 
 	assert.Equal(t, now, item.ExpiresAt())
 }
+
+func Test_Item_Version(t *testing.T) {
+	// TTL=DefaultTTL
+	item := newItem("key1", "value1", DefaultTTL, true)
+	assert.Equal(t, int64(0), item.Version())
```

Comment:
A simple
```go
item := Item{version: 10}
assert.Equal(t, 10, item.Version())
```

should be enough to test this. Again, it's better not to use `newItem` or `item.update()` to properly isolate the test.